### PR TITLE
CX-14789 : Removed role and aria labels for disabled fields

### PR DESCRIPTION
**Pull request by niti-owner**
## Description
CX-14789

## Screenshots:
**Before** (If applicable):

**After**:

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: web-components/src/components/editable-textfield/EditableTextfield.ts
Comment (mithun-srinivas) [inline web-components/src/components/editable-textfield/EditableTextfield.ts:1]: if the textbox is disabled, still the screen reader should the placeholder and say its disabled or dimmed [voiceover] right ?

Comment (niti-owner) [reply web-components/src/components/editable-textfield/EditableTextfield.ts:1]: Yes, you are right, I will discuss this point with Abhishek and Applause team if required. Without this code change it announces dimmed text. Which should be ok. Did this change as the story is expecting that role, and aria labels should be removed if the field is not editable. But I am not sure if that is right thing to do either.

Comment (niti-owner) [reply web-components/src/components/editable-textfield/EditableTextfield.ts:1]: Got it reviewed by UX. Comments: https://jira-eng-sjc12.cisco.com/jira/browse/CX-14789

**Review by Samreen26 — DISMISSED** [review]

**Review by mithun-srinivas — APPROVED** [review]

**Review by vinitkumargoel — DISMISSED** [review]

**Review by vinitkumargoel — COMMENTED** [review]
Can you add unit test case for all of these

**Review by vinitkumargoel — APPROVED** [review]

**Pull request merged**